examples/aws-atlas-privatelink-regionalized/aws-west-vpc.tf

```diff
@@ -0,0 +1,56 @@
+# Create Primary VPC
+resource "aws_vpc" "west" {
+  provider             = aws.west
+  cidr_block           = "10.0.0.0/16"
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+}
+
+# Create IGW
+resource "aws_internet_gateway" "west" {
+  provider = aws.west
+  vpc_id   = aws_vpc.west.id
+}
+
+# Route Table
+resource "aws_route" "west-internet_access" {
+  provider               = aws.west
+  route_table_id         = aws_vpc.west.main_route_table_id
+  destination_cidr_block = "0.0.0.0/0"
+  gateway_id             = aws_internet_gateway.west.id
+}
+
+# Subnet-B
+resource "aws_subnet" "west" {
+  provider                = aws.west
+  vpc_id                  = aws_vpc.west.id
+  cidr_block              = "10.0.1.0/24"
+  map_public_ip_on_launch = true
+  availability_zone       = "${var.aws_region_west}b"
+}
+
+/*Security-Group
+Ingress - Port 80 -- limited to instance
+          Port 22 -- Open to ssh without limitations
+Egress  - Open to All*/
```

Comment:
Kind of a nit but this isn't what the security group sets up below - it's open both ingress and egress.  If we do update this we should probably note that's not recommended but just an example ;)

Reply:
I believe this has been fixed by updating the actual configuration in the example, but let me know if I missed the boat

Reply:
I think SSH is still missing, and you probably just to to 80, not from 80 but in reality all we need to make sure of is anything is here that is needed to setup this example and neither 22 or 80 are needed.  So I think this is okay for now and maybe we remove the comment later and/or make sure it' matches 100%.   I'm calling this no blocking.